optout/src/main/java/gov/cms/ab2d/optout/SpringBootApp.java

```diff
@@ -0,0 +1,18 @@
+package gov.cms.ab2d.optout;
```

Comment:
Take a look at the `bfd` or `hpms` modules. They don't have `SpringBootApp` under `src/main` because those modules are technically not executable Boot apps, but they _do_ have it under `src/test` to help with wiring in tests. We should probably follow the same approach.

Reply:
moved `SpringBootApp` to `src/test/java`